src/printer.cc

```diff
@@ -136,34 +137,10 @@ std::string Printer::Stringify(v8::JSFunction js_function, Error& err) {
 
 template <>
 std::string Printer::Stringify(v8::JSDate js_date, Error& err) {
-  std::string pre = "<JSDate: ";
-
-  v8::Value val = js_date.GetValue(err);
-
-  v8::Smi smi(val);
-  if (smi.Check()) {
-    std::string s = smi.ToString(err);
-    if (err.Fail()) {
-      return pre + ">";
-    }
-
-    return pre + s + ">";
-  }
-
-  v8::HeapNumber hn(val);
-  if (hn.Check()) {
-    std::string s = hn.ToString(true, err);
-    if (err.Fail()) {
-      return pre + ">";
-    }
-    return pre + s + ">";
-  }
-
-  double d = static_cast<double>(val.raw());
```

Comment:
@evanlucas do you remember why this last part, casting to `double` was added? I couldn't find any test case where I could reach it. It was originally introduced in https://github.com/nodejs/llnode/pull/3/files

Reply:
not off the top of my head, sorry!